IBAnimatable/PresentationPresenterChacheKey.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  PresentationChacheKey.swift
+//  IBAnimatable
+//
+//  Created by Miro on 8/12/16.
+//  Copyright © 2016 IBAnimatable. All rights reserved.
+//
+
+import Foundation
+
+struct PresentationPresenterChacheKey: Hashable {
```

Comment:
Can we directly use `PresentationAnimationType` enum as the key, and let `PresentationAnimationType: Hashable`  and provide ` static func == (lhs: PresentationPresenterChacheKey, rhs: PresentationPresenterChacheKey)`? If we can, then we don't need the additional struct. enum is also a value type.